packages/mdc-switch/foundation.js

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
Update README for MDCSwitchFoundation & MDCSwitchAdapter.

Reply:
Good catch! Updated.